app/home/-components/hero-banner/component.ts

```diff
@@ -18,20 +19,22 @@ const { featureFlagNames: { ABTesting } } = config;
 
 @layout(template, styles)
 @tagName('')
-export default class NewHome extends Component {
+export default class NewHomeHeroBanner extends Component {
+    @service analytics!: Analytics;
     @service features!: Features;
 
     @alias(`features.${camelize(ABTesting.homePageVersionB)}`)
     shouldShowVersionB!: boolean;
 
     @computed('shouldShowVersionB')
-    get versionClass(this: NewHome): string {
-        return this.shouldShowVersionB ? 'versionB' : '';
+    get version(this: NewHomeHeroBanner): string {
+        return this.shouldShowVersionB ? 'versionB' : 'versionA';
     }
 
     @action
     search(query: string) {
         const { search } = serviceLinks;
+        this.analytics.track('search', 'enter', `Logged-out homepage ${this.version} - Search`);
```

Comment:
Looks like this is double-tracked: once here, and [once in `<SearchBar>`](https://github.com/CenterForOpenScience/ember-osf-web/pull/705/files#diff-73d9d27cc3edc662952a68dd6c679848R20). I'd vote for removing this one, so `<SearchBar>` handles its own analytics.